test/browser_tests/page_objects/page_blog-single.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+function BlogSingle() {
+
+  this.get = function() {
+    browser.get( '/blog/lessons-weve-learned/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/f18474a240fd0f56bdd225730736293f2f28c11c